graphx/src/test/scala/org/apache/spark/graphx/lib/PageRankSuite.scala

```diff
@@ -321,7 +321,7 @@ class PageRankSuite extends SparkFunSuite with LocalSparkContext {
         val rank = if (vid < source) {
           0.0
         } else {
-          a * Math.pow(1 - resetProb, vid - source)
+          a * Math.pow(1.0 - resetProb, vid.toDouble - source)
```

Comment:
nit: `resetProb` is already a double type, it seems unnecessary to explicitly change it to 1.0, right?

Reply:
Done!